### ISSUE TITLE
Provide a way to log file into test

### ISSUE THREAD

**Issue by nvborisenko** [reportportal/commons-net#43]
Currently user may use:
```csharp
any_log_framework.Info("my dog {rp#file#" + filePath + "}")` // and then log framework appender tries to conver it to known message
```

At the same time we have:
``` csharp
ReportPortal.Shared.Log.Info("my message")
```

What if we have overloaded method to accept `FileInfo`?
```csharp
ReportPortal.Shared.Log.Info(new FileInfo(<path_to_the_file>))
```

**Comment by nvborisenko** [comment]
This is easier than https://github.com/reportportal/commons-net/issues/27

**Comment by Aleh-Yanushkevich** [comment]
@nvborisenko  Is it still relevant? Because I see overloaded method like:
![image](https://user-images.githubusercontent.com/103627305/173191698-8f9df17f-99d8-40b2-82a1-dd42aeb49030.png)

**Comment by nvborisenko** [comment]
yes, we can do something like
```csharp
void Info(string message, System.IO.FileInfo file)
```

**Comment by nvborisenko** [comment]
Resolved in scope of #87
